src/Helper/Quote.php

```diff
@@ -158,13 +158,11 @@ protected function setItemQty()
      */
     protected function setPayment()
     {
-        $paymentModel = $this->paymentFactory->getPayment($this->sgBase->getPaymentMethod())->getPaymentModel();
+        $defaultPayment = $this->paymentFactory->getPayment(strtolower(PaymentFactory::DEFAULT_PAYMENT_METHOD));
```

Comment:
i keep seeing this `strtolower` and upper stuff. I wonder if there is a nice way to handle it inside functions. Or should we?